scrooge-generator/src/main/scala/com/twitter/mustache/ScalaObjectHandler.scala

```diff
@@ -0,0 +1,69 @@
+package com.twitter.mustache
```

Comment:
sorry if I'm dense.. is this class only used by scrooge generator? Not sure how a class with a different name that's used by a generator would conflict with something.